### PR TITLE
updated getMesh JS binding

### PR DESCRIPTION
**Pull request by elalish**
Now `getMesh` returns a mesh of the same form that can be used to produce a new manifold with `smooth`.

### CODE REVIEW

File: bindings/wasm/bindings.cpp [hunk 1]
```diff
@@ -169,9 +169,9 @@ EMSCRIPTEN_BINDINGS(whatever) {
       .function("add", &Union)
       .function("subtract", &Difference)
       .function("intersect", &Intersection)
-      .function("getMesh", &Manifold::GetMesh)
+      .function("_GetMesh", &Manifold::GetMesh)
       .function("refine", &Manifold::Refine)
-      .function("_Warp", &Warp, allow_raw_pointers())
+      .function("_Warp", &Warp)
```
Comment (pca006132) [inline bindings/wasm/bindings.cpp:174]: `allow_raw_pointers` is not needed now? Is it documented in emscripten documentation?

**Review by pca006132 — APPROVED** [review]
Other than that LGTM

File: bindings/wasm/bindings.cpp (hunk 1 above)
Comment (elalish) [reply bindings/wasm/bindings.cpp:174]: Actually, I noticed you didn't add it when you made `levelSet`, which uses the same thing; I believe you changed the pointer type from `void *`. So I figured I wouldn't need it here anymore either, and sure enough, it works.

Comment (pca006132) [reply bindings/wasm/bindings.cpp:174]: Oh OK, good to know

**Pull request merged**